### DISCUSSION TITLE
How to find out with what rights a commit was made?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22478] [New to GitHub]
<p>SUBJ<br>
I found a commit in my private repository that i didn’t do</p>

**Comment by jfullstackdev** [comment]
<p>can you send the link of that specific commit ?</p>

**Comment by ghost** [comment]
<p><a href="https://github.com/evodefi/bridge-ui/commit/73b554bab9a56d39eb691c88b8b82e1c1fbaa15d" class="onebox" target="_blank" rel="noopener nofollow ugc">https://github.com/evodefi/bridge-ui/commit/73b554bab9a56d39eb691c88b8b82e1c1fbaa15d</a></p>

**Comment by jfullstackdev** [comment]
<p>that’s 404</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/f/0ffaa92d9dbe1b8ad8945d4dd03caa6600250ee3.jpeg" data-download-href="/uploads/short-url/2hmcqyh2YZqGbmFRLbLrzbHj3N1.jpeg?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181071339-57a29251-9be5-439e-853f-3c3f422a5203.jpeg" alt="image" data-base62-sha1="2hmcqyh2YZqGbmFRLbLrzbHj3N1" width="575" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/f/0ffaa92d9dbe1b8ad8945d4dd03caa6600250ee3_2_575x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/0/f/0ffaa92d9dbe1b8ad8945d4dd03caa6600250ee3_2_862x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/0/f/0ffaa92d9dbe1b8ad8945d4dd03caa6600250ee3_2_1150x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/f/0ffaa92d9dbe1b8ad8945d4dd03caa6600250ee3_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1198×1040 186 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ghost** [comment]
<p>Because this is private repository. I heed to github administation help</p>

**Comment by jfullstackdev** [comment]
<p>how about screenshot ?</p>

**Comment by ghost** [comment]
<p>I don’t think a screenshot will help here. How can I contact github support?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>here,</p>
<p><a href="https://support.github.com/contact/account">https://support.github.com/contact/account</a></p>
